airflow/executors/kubernetes_executor.py

```diff
@@ -52,6 +52,7 @@
 from airflow.utils.state import State
 
 ALL_NAMESPACES = "ALL_NAMESPACES"
+POD_DONE_KEY = "airflow_done"
```

Comment:
Happy to adjust naming if there are better ideas.

Reply:
just to shed it up...

"executor_done" or "do_not_adopt"

Reply:
Thanks, renamed some stuff based on this. Take another look please 🙏.